.github/workflows/build.yaml

```diff
@@ -71,7 +76,7 @@ jobs:
     - name: Install dependencies
       run: make setup
     - name: Create changelog and bump
-      if: github.ref == 'refs/heads/main' && matrix.os == env.ARTIFACT_OS && matrix.python == env.ARTIFACT_PYTHON
+      if: ${{ matrix.os == 'ubuntu-latest' && matrix.python == '3.10' && !startsWith(github.event.commits[0].message, 'bump:') }}
```

Comment:
Dang! I thought that `if:` interprets its value as JS?

Reply:
> Dang! I thought that `if:` interprets its value as JS?

It does, but notice the `!` which is a reserved symbol in YAML, hence using `${{}}`.

Reply:
> And you wired the OS and Python, instead of using the variables — only here, and not in other lines?

:facepalm:

Reply:
Why do I keep asking the same question 🤦🏻‍♂️